bodhi/server/static/js/update_form.js

```diff
@@ -200,9 +205,14 @@ $(document).ready(function() {
                 }
                 $.each(data.rows, function(i, bug) {
                     // Insert the checkbox only if this ID is not already listed
-                    if ($.inArray(bug.id, checked_bug_ids) == -1) {
-                        add_bug_checkbox(bug.id, bug.description, false);
-                    }
+                    var listed = false;
+                    $.each(checked_bugs, function(i, checked_bug) {
+                        if (bug.id == checked_bug.id) {
+                            listed = true;
+                            return;
```

Comment:
Since you have a ```return``` here, I wonder if we can get away without having the ```listed``` boolean.

Reply:
The problem here is that it seems there's no way to break/continue the outer $.each loop, so I need to use a boolean var.
Anyway I realized the `return` statement is wrong, since it translates in `continue`, while I should have used `return false` to break the inner loop here. I'm going to change this in `break` to be clearer.